app/tests/unit/specs/components/common/PageFeatureNotAvailable.spec.js

```diff
@@ -1,14 +1,30 @@
-import { mount } from "@vue/test-utils"
+import { mount, createLocalVue } from "@vue/test-utils"
 import PageFeatureNotAvailable from "common/PageFeatureNotAvailable"
+import Vuex from "vuex"
+const localVue = createLocalVue()
+
+localVue.use(Vuex)
 
 describe(`PageFeatureNotAvailable`, () => {
   let wrapper
+  let store
+
   beforeEach(() => {
+    store = new Vuex.Store({
```

Comment:
feels funny to me that a store is required to test this — but c'est la vie

Reply:
we don't need to init a store here, right? it can just be n object as in all the other tests.

Reply:
can you share an example of what you mean?

Reply:
check other tests for comparison. we never instantiate a store in a test

Reply:
yeah we do in 3 places - that's where i copied it from